node/src/Commands.ts

```diff
@@ -2421,6 +2421,132 @@ export function createZIncrBy(
     ]);
 }
 
+/**
+ * Optional arguments to {@link GlideClient.sort|sort}, {@link GlideClient.sortStore|sortStore} and {@link GlideClient.sortReadOnly|sortReadOnly} commands.
+ *
+ * See https://valkey.io/commands/sort/ and https://valkey.io/commands/sort_ro/ for more details.
```

Comment:
https://valkey.io/commands/sort_ro/ is currently unavailable. Do we want to add this link now?